tests/suites/test_suite_ssl.function

```diff
@@ -14,8 +14,9 @@
 
 /* Mnemonics for the early data test scenarios */
 #define TEST_EARLY_DATA_ACCEPTED 0
-#define TEST_EARLY_DATA_DEPROTECT_AND_DISCARD 1
-#define TEST_EARLY_DATA_DISCARD_AFTER_HRR 2
+#define TEST_EARLY_DATA_NO_INDICATION_SENT 1
+#define TEST_EARLY_DATA_DEPROTECT_AND_DISCARD 2
```

Comment:
Note: this name an the next one are a bit server-centric. Perhaps "not accepted" and "rejected hrr" would be more neutral names?

Edit: done in the next two commits :)